src/drt/src/pa/FlexPA_graphics.cpp

```diff
@@ -110,50 +110,56 @@ void FlexPAGraphics::drawLayer(odb::dbTechLayer* layer, gui::Painter& painter)
     return;
   }
 
-  layerNum = layer_map_.at(layer->getNumber()).first;
-  if (layerNum < 0) {
+  layer_num = layer_map_.at(layer->getNumber()).first;
+  if (layer_num < 0) {
     return;
   }
 
   for (auto via : pa_vias_) {
     auto* via_def = via->getViaDef();
-    Rect bbox;
+    Rect boundary_box;
```

Comment:
bbox is a common term in EDA but if you spell it out it would be bounding_box

Reply:
Okay, revert boundary_box to bbox then